src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HandleWrapper.h

```diff
@@ -9,7 +9,7 @@
 struct InvalidHandleTraits
 {
     using HandleType = HANDLE;
-    static const HANDLE DefaultHandle;
+    static constexpr HANDLE DefaultHandle = nullptr;
```

Comment:
Being optimistic that the [compiler bug that prevented this](https://developercommunity.visualstudio.com/content/problem/33928/constexpr-failing-on-nullptr-v141-compiler-regress.html) is fixed now. We'll see :).